### PR TITLE
ME-14382-lazy-shoppable-plugin

### PR DESCRIPTION
**Pull request by tsi**
This PR wraps the shoppable plugin with a lazy loading wrapper

### CODE REVIEW

File: src/plugins/shoppable-plugin/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default async function lazyPlugin(player, options) {
```
Comment (ehab-cl) [inline src/plugins/shoppable-plugin/index.js:1]:
```suggestion
export default async function lazyShoppablePlugin(player, options) {
```

File: src/video-player.js [hunk 2]
```diff
@@ -533,9 +533,8 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       return this.videojs.cloudinary.source(publicId, options);
     }
 
-    // Interactive plugin - available in full (not light) build only
-    if (this.videojs.interactive) {
-      this.videojs.interactive(this.videojs, options);
```
Comment (ehab-cl) [inline src/video-player.js:538]: why are we removing these 2 lines?

**Review by ehab-cl — DISMISSED** [review]
💪

File: src/video-player.js (hunk 2 above)
Comment (tsi) [reply src/video-player.js:538]: The "interactive" plugin was supposed to wrap the shoppable plugin as well as other relevant features, this never happened and it only contains the shoppable plugin.
Meanwhile, the interaction-area plugin was introduced, so this just became even more confusing.
Bottom line - the "interactive" plugin became "shoppable"

**Review by jakub-roch — APPROVED** [review]

**Pull request merged**